src/Microsoft.DotNet.Build.CloudTestTasks/Microsoft.DotNet.Build.CloudTestTasks.csproj

```diff
@@ -20,6 +20,12 @@
     <StartupObject />
   </PropertyGroup>
   <ItemGroup>
+    <Compile Include="..\Microsoft.DotNet.Build.Tasks\ZipFileCreateFromDependencyLists.cs">
+      <Link>ZipFileCreateFromDependencyLists.cs</Link>
+    </Compile>
+    <Compile Include="..\Microsoft.DotNet.Build.Tasks\ZipFileCreateFromDirectory.cs">
+      <Link>ZipFileCreateFromDirectory.cs</Link>
+    </Compile>
```

Comment:
Does it make sense to put these in `$(CommonPath)` to make it more obvious they're shared?